File: Python/Easy/two_digit_sum.py
# https://edabit.com/challenge/KT8ApJ2EJcLz4K3t2
def two_digit_sum(n):
    sum = 0
    while n != 0:
        sum+=n%10
        n//=10
    return sum
#
#
print(two_digit_sum(45), 9)

File: javascript/Very_Easy/twoDigitSum.js
// https://edabit.com/challenge/M5RTAawPtRSByE3HT

function twoDigitSum(n) {
	return n%10+Math.floor(n/10)
}


console.log(twoDigitSum(45), 9)
console.log(twoDigitSum(65), 11)